ledger/src/blockstore_db.rs

```diff
@@ -2310,7 +2303,7 @@ mod rocks_metrics_utils {
 
     /// Reports the collected PerfContext and disables the PerfContext after
     /// reporting.
-    pub fn report_read_perf_context(metric_header: &'static str) {
+    pub(crate) fn report_read_perf_context(metric_header: &'static str) {
```

Comment:
again why `pub(crate)`?
The function seems to be private to this module.

Reply:
Same as the above comment.  It belongs to mod `rocks_metrics_utils`.